autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionSelect.css

```
@@ -71,10 +75,68 @@
     margin-top: 8px;
 }
 
+.indent-for-checkbox:global(.MuiListItem-root) {
+    padding-left: 14px;
+}
+
+.name-honors-icon :global(.MuiButton-label) {
+    justify-content: normal;
+}
+
 .no-start-padding {
     padding-inline-start: 0;
 }
 
+.meeting-info-wrapper {
+    display: contents;
+}
+
+@media (max-width: 1600px) {
+    .section-details-table {
+        grid-template-columns: 1fr 1fr;
+    }
+    .meeting-info-wrapper > div:nth-child(2) {
+        text-align: right; /* applies to building in 2-column view */
+    }
+    .meeting-info-wrapper > div:nth-child(3) {
+        text-align: left; /* applies to days in 2-column view */
+    }
+}
+
+.meeting-info-wrapper:nth-child(2n) div {
```

```suggestion
.meeting-info-wrapper:nth-child(even) div {
```
I don't actually care if you make this change since this is perfectly readable but just so you know, there are `nth-child(even)` and `nth-child(odd)` selectors